jenkins/swift-functional-tests/10-install-ring.sh

```diff
@@ -1,12 +1,28 @@
 #!/bin/bash -xue
 
+function get_AllowEncodedSlashes {
```

Comment:
Hmm, docs says that slashes are not quoted by default : https://docs.python.org/2/library/urllib.html, so you won't have any encoded slash reaching sproxyd front http server.
Anyway if it does not hurt to have that code in the 0.3 job , even though no needed, then it's your call !